src/deps/freetype/src/autofit/afglobal.c

```diff
@@ -331,10 +331,11 @@
     {
       af_face_globals_free( globals );
       globals = NULL;
+    } else
```

Comment:
This whole file is just part of an external dependency that will be updated from time to time. It would be better to send this change to the original authors